extensions/database.py

```
@@ -78,37 +70,30 @@ def session(Sid: bytes, Uid: int) -> None:
-            request(
-                "SELECT Uid FROM SESSIONS WHERE Sid = ?",
-                [
-                    SUG.REQ.SID
-                ]
-            )[0]["Uid"]
+            SUG.REQ.SID
         ]
-    )[0]
\ No newline at end of file
+    ))
+    if SUG.THR.DBV: 
```

Ruff (0.11.9)

`SUG` may be undefined, or defined from star imports

(F405)